examples/v2/chat2.nim

```diff
@@ -303,6 +307,13 @@ proc processInput(rfd: AsyncFD, rng: ref BrHmacDrbgContext) {.async.} =
 
     await node.query(HistoryQuery(contentFilters: @[HistoryContentFilter(contentTopic: DefaultContentTopic)]), storeHandler)
 
+  if conf.swap:
+    echo "Fetching account state"
```

Comment:
See my comment on using `echo` elsewhere. Only applications designed for interaction with stdout should use `echo`. This could probably be a `debug` level log.